.bazelrc

```diff
@@ -4,14 +4,6 @@
 # Mandatory at the moment for Bazel 7.0.0.
 build --noincompatible_sandbox_hermetic_tmp
 
-# This is mandatory if your build env doesn't have a proper Python 3 exe
```

Comment:
workspace-based workflow will remain as default, as that's the one fully functional.

I realized we need good documentation now to navigate new users and less experienced with Bazel: workspace vs bzlmod. Default workspace and clearly state that all advertised features are functional. Then bzlmod with decent notes on caveats.

Reply:
Both variants are fully functional and these .bazelrc lines are not needed in either anymore